.changeset/cyan-ears-wonder.md

```diff
@@ -0,0 +1,50 @@
+---
+'merchant-center-application-template-starter': minor
+'@commercetools-frontend/mc-scripts': minor
+'playground': minor
+'@commercetools-website/custom-applications': minor
+---
+
+New CLI options! 🎉
+
+## Loading dotenv files
+
+The `mc-script` CLI now supports loading environment variables from [dotenv files](https://www.npmjs.com/package/dotenv).
```

Comment:
That's great to have now as it is best practice nowadays 👍  
Ten years of commercetools experience in this PR 🎉 
@emmenko Thanks for all your contributions. Keep it up!